modules/tinymce/src/core/test/ts/browser/DragnDropCEFTest.ts

```diff
@@ -0,0 +1,82 @@
+import { Mouse, UiFinder } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { Arr } from '@ephox/katamari';
+import { SugarLocation } from '@ephox/sugar';
+import { TinyAssertions, TinyDom, TinyHooks } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+
+describe('browser.tinymce.core.DragnDropCEFTest', () => {
+  const getBaseCEFElement = (name: string) => [ `<div class="${name}" style="margin: 40px;" contenteditable="false">`,
+    '<div style="height: 20px; cursor: grab; background-color: lightgrey;">&nbsp;',
+    '</div>',
+    '<div style="border: 1px solid lightgrey;">',
+    '<div>',
+    '<div style="padding-left: 50px;">',
+    `<label contenteditable="true">${name} header</label>`,
+    '</div>',
+    '<div>',
+    '<article style="padding: 10px 50px;" contenteditable="true">',
+    `<p>${name} description</p>`,
+    '</article>',
+    '</div>',
+    '</div>',
+    '</div>',
+    '</div>'
+  ].join('');
+
+  const getContentWithCefElements = (elementsNames: string[]): string => `<div>${Arr.foldl(elementsNames, (acc, elementName) =>
```

Comment:
A fold inside a template string that is a bit hard to read at least for me. Maybe just pull out the fold result into a variable and use that in the template string.

Reply:
simplifying it I removed the `join` and now is just a template string